airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityTest.java

```diff
@@ -290,9 +290,9 @@ public void setAttemptFailureWrapException() throws IOException {
     public void setJobCancelled() throws IOException {
       jobCreationAndStatusUpdateActivity.jobCancelled(new JobCancelledInput(JOB_ID, ATTEMPT_ID, failureSummary));
 
-      Mockito.verify(mJobPersistence).cancelJob(JOB_ID);
       Mockito.verify(mJobPersistence).failAttempt(JOB_ID, ATTEMPT_ID);
       Mockito.verify(mJobPersistence).writeAttemptFailureSummary(JOB_ID, ATTEMPT_ID, failureSummary);
+      Mockito.verify(mJobPersistence).cancelJob(JOB_ID);
```

Comment:
Nit, is it testing the order or should we add is explicitely using InOrder: https://stackoverflow.com/questions/21901368/mockito-verify-order-sequence-of-method-calls

Reply:
Wasn't aware of that, thanks for the suggestion! I added that to this test